section2/reflection.md

```diff
@@ -2,14 +2,38 @@
 
+    I am demonstrating a growth mindset because I am challenging myself to learn a new language, coding. I am pushing myself out of my comfort zone and trying new things, experimenting with code and learning from mistakes. When I am making mistakes, I persevere to figure it out and not give up.
+
+    One way that I am not demonstrating a growth mindset is being hesitant to ask questions. Asking for help is not my strength, but I am cognizant of this and am working on it to improve.
+
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
 
+    I liked the challenge of having to remember what I have previously learned, or go back to my notes/research google for a reminder or to fill in blanks.
+  This required a little more problem-solving skills for me, which could be uncomfortable at times, but having resources available made this very doable.
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
+    A conditional statement evaluates to true or false. Comparison operators are the most common operators used for conditions, but logical operators can also be used for boolean data
+
+    Daily life conditional statement: If the weather is sunny, then wear shorts, else if weather is rainy, take an umbrella.
+
+    I would think that conditional statements are used on any form that is filled out online, where if all blanks have input, then able to submit form. Else redirect to empty blanks that still need input.
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+    Multiple conditions can be used with an if statement using "else if." In other words, if the first condition is not met, else if is used with second condition. If second condition is not met, then move to else if with third condition.
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+    Logical operators can be used to evaluate whether 2 conditions have been met (&& = AND), or even if one of the two (|| = OR).These logical operators can be plugged into an if or if/else statement.
+
+7. What questions do you still have about `if` statements and/or functions?
+
+    Is it possible to combine the responses to logical statements into a single function, or evaluate for a combination of responses rather than typing out the && or || statement every time?
```

Comment:
There may be some scenarios where you encapsulate that logic somewhere else... but either way, at the end of the day, if you want 2 things checked, you've got to be explicit in the directions you give to the JS interpreter.